.travis.yml

```diff
@@ -3,9 +3,7 @@ node_js: node
 install:
     - npm install
     - export PATH=$HOME/.local/bin:$PATH
-    - pip install -r requirements.txt --user `whoami`
 script:
```

Comment:
To fix the Travis build, you'll probably just need to run `npm run-script build` before `npm test`.

Reply:
Yes